Corona-Warn-App/src/main/res/navigation/nav_graph.xml

```diff
@@ -247,6 +247,25 @@
             android:name="symptoms"
             app:argType="de.rki.coronawarnapp.submission.Symptoms" />
     </fragment>
+    <fragment
+        android:id="@+id/submissionResultPositiveOtherWarningNoConsentFragment"
+        android:name="de.rki.coronawarnapp.ui.submission.warnothers.SubmissionResultPositiveOtherWarningNoConsentFragment"
+        android:label="fragment_submission_result_positive_other_no_consent_warning"
+        tools:layout="@layout/fragment_submission_no_consent_positive_other_warning">
+        <argument
+            android:name="symptoms"
+            app:argType="de.rki.coronawarnapp.submission.Symptoms" />
+        <action
+            android:id="@+id/action_submissionResultPositiveOtherWarningNoConsentFragment_to_mainFragment"
+            app:destination="@id/mainFragment"
+            app:popUpTo="@id/mainFragment"
+            app:popUpToInclusive="true" />
+        <action
+            android:id="@+id/action_submissionResultPositiveOtherWarningNoConsentFragment_to_submissionDoneFragment"
+            app:destination="@id/submissionDoneFragment"
+            app:popUpTo="@id/submissionDoneFragment"
```

Comment:
You need to pop to homeFragment ( Inclusive = false) here to clear the back stack

Reply:
Done